server/@types/ui/index.d.ts

```diff
@@ -87,6 +87,13 @@ type ReferralSharedPageData = {
   submissionSummaryListRows: Array<GovukFrontendSummaryListRowWithKeyAndValue>
 }
 
+type ReferralSummaryDetails = {
+  personName: Person['name']
+  personPrisonNumber: Person['prisonNumber']
```

Comment:
We might be able to use the backend type here once it's been flattened, if we're not needing to fetch `personName` in the UI.